portal/public/js/mojFrontend.js

```diff
@@ -1,3 +1,3 @@
.test(n)?j(t).position()[e]+"px":n}))})),j.each({Height:"height",Width:"width"},(function(t,e){j.each({padding:"inner"+t,content:e,"":"outer"+t},(function(n,o){j.fn[o]=function(r,i){var a=arguments.length&&(n||"boolean"!=typeof r),s=n||(!0===r||!0===i?"margin":"border");return tt(this,(function(e,n,r){var i;return y(e)?0===o.indexOf("outer")?e["inner"+t]:e.document.documentElement["client"+t]:9===e.nodeType?(i=e.documentElement,Math.max(e.body["scroll"+t],i["scroll"+t],e.body["offset"+t],i["offset"+t],i["client"+t])):void 0===r?j.css(e,n,s):j.style(e,n,r,s)}),e,a?r:void 0,a)}}))})),j.each(["ajaxStart","ajaxStop","ajaxComplete","ajaxError","ajaxSuccess","ajaxSend"],(function(t,e){j.fn[e]=function(t){return this.on(e,t)}})),j.fn.extend({bind:function(t,e,n){return this.on(t,null,e,n)},unbind:function(t,e){return this.off(t,null,e)},delegate:function(t,e,n,o){return this.on(e,t,n,o)},undelegate:function(t,e,n){return 1===arguments.length?this.off(t,"**"):this.off(e,t||"**",n)},hover:function(t,e){return this.mouseenter(t).mouseleave(e||t)}}),j.each("blur focus focusin focusout resize scroll click dblclick mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave change select submit keydown keypress keyup contextmenu".split(" "),(function(t,e){j.fn[e]=function(t,n){return arguments.length>0?this.on(e,null,t,n):this.trigger(e)}}));var on=/^[\s\uFEFF\xA0]+|([^\s\uFEFF\xA0])[\s\uFEFF\xA0]+$/g;j.proxy=function(t,e){var n,o,r;if("string"==typeof e&&(n=t[e],e=t,t=n),v(t))return o=s.call(arguments,2),r=function(){return t.apply(e||this,o.concat(s.call(arguments)))},r.guid=t.guid=t.guid||j.guid++,r},j.holdReady=function(t){t?j.readyWait++:j.ready(!0)},j.isArray=Array.isArray,j.parseJSON=JSON.parse,j.nodeName=A,j.isFunction=v,j.isWindow=y,j.camelCase=rt,j.type=k,j.now=Date.now,j.isNumeric=function(t){var e=j.type(t);return("number"===e||"string"===e)&&!isNaN(t-parseFloat(t))},j.trim=function(t){return null==t?"":(t+"").replace(on,"$1")},void 0===(n=function(){return j}.apply(e,[]))||(t.exports=n);var rn=o.jQuery,an=o.$;return j.noConflict=function(t){return o.$===j&&(o.$=an),t&&o.jQuery===j&&(o.jQuery=rn),j},void 0===r&&(o.jQuery=o.$=j),j}))}},e={};function n(o){var r=e[o];if(void 0!==r)return r.exports;var i=e[o]={exports:{}};return t[o].call(i.exports,i,i.exports,n),i.exports}n.n=function(t){var e=t&&t.__esModule?function(){return t.default}:function(){return t};return n.d(e,{a:e}),e},n.d=function(t,e){for(var o in e)n.o(e,o)&&!n.o(t,o)&&Object.defineProperty(t,o,{enumerable:!0,get:e[o]})},n.o=function(t,e){return Object.prototype.hasOwnProperty.call(t,e)},n.r=function(t){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(t,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(t,"__esModule",{value:!0})};var o={};!function(){"use strict";n.r(o);var t=n(755),e=n.n(t),r=n(722),i=n.n(r);o.default=(window.$=e(),(0,r.initAll)(),void(window.MOJFrontend=i()))}(),(DTFS_PORTAL=void 0===DTFS_PORTAL?{}:DTFS_PORTAL).mojFrontend=o}();
```

Comment:
Out of curiosity, why do the other 2 UIs not have this file?

Reply:
Had a chat with Abhi about this. TFM UI does have this file, and Abhi rebuilt the file but there were no changes. GEF does not have this file because it uses the file from portal